src/l10n/en.ts

```diff
@@ -6,5 +6,10 @@ export const en = {
   preferredTerm: 'Preferred term',
   fieldOfStudy: 'Field of study',
   validity: 'Validity',
-  status: 'Status'
+  status: 'Status',
+  organisationNumber: 'Organization number:',
+  information: 'Information',
+  informationText: 'This is a collaborative project between Brønnøysundregistrene, Difi and SKATE government agencies.',
+  mail: 'fellesdatakatalog@brreg.no',
+  privacy: ' and privacy'
```

Comment:
Hvorfor trenger du denne som egen key, kan den ikke stå sammen med 'information'? Hvis du må bygge opp setninger så kan det også gjøres dynamisk:

nb.js:
[exampleText]: 'Denne listen inneholder {0} elementer'

f.eks. localization.formatString(
        localization.['exampleText'],
        '10'
      )

=> 'Denne listen inneholder 10 elementer'.

Reply:
Egentlig trenger vi ingen dynamisk tekst i footer, kunne vell bare vært:

privacy: "Informasjonskapsler og personvern"

og 

privacy: "Information capsules and privacy"

Usikker på hvorfor det er gjort slik, men lar det stå.